apis/spec/v1beta1/clusterworkloadresourcemapping_webhook.go

```diff
@@ -145,3 +139,22 @@ func verifyJsonPath(node jsonpath.Node, path *field.Path, value string) field.Er
 	}
 	return errs
 }
+
+func (mapping *ClusterWorkloadResourceMapping) AcceptsGVR(gvk *schema.GroupVersionResource) bool {
+	expectedName := fmt.Sprintf("%v.%v", gvk.Resource, gvk.Group)
+	if mapping.Name != expectedName {
+		fmt.Printf("expected name: %s\ngot: %s\n", expectedName, mapping.Name)
```

Comment:
There is logger initialized in this package: https://github.com/redhat-developer/service-binding-operator/blob/a6d68cb7250229685d529758c05d50c560b655df/apis/spec/v1beta1/servicebinding_webhook.go#L30

Reply:
+1

Reply:
I've taken this printf out in 3019f3b3; I don't think it makes sense to log when this test fails.

I originally had this printf in there for debugging purposes; it should no longer be needed.